common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayWork.scala

```diff
@@ -179,7 +173,7 @@ case object DisplayWork {
         if (includes.images)
           Some(work.data.images.map(DisplayWorkImageInclude(_)))
         else None,
-      ontologyType = displayWorkType(work.data.workType),
+      ontologyType = DisplayWorkType(work.data.workType),
```

Comment:
I'm not sure that this should be a whole class of its own as it's just a string and so is a bit confusing to read - that's why I didn't do it like this the first time round

Reply:
It needs to be used in a few places hence why I split it out

Reply:
Generally don't really like an overloaded `apply` that returns a type other than the class (even when it's a singleton / object), and even more so when that type is a primitive

Reply:
Yeh I get you. I am not bothered about it too much personally, I guess I am viewing `DisplayWorkType` as an alias for `String` here, I could be more explicit and add the type alias but scala is annoying in that it doesn't allow type alias at top level.